<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task_Tab</class>
 <widget class="QWidget" name="Task_Tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(37, 40, 50);
	color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QTabWidget::pane{
border-color:none;
}
QTabWidget:tab-bar {

border-color:none;
}
QTabBar:tab:selected{
background-color:#282c37;
color:white;
}
QTabBar:tab:!selected{
background-color:#1b1d22;
color:white;
}

QLineEdit{
	border: 0px solid rgb(37, 39, 48);
	height: 30px;
	border-radius: 15px;
	color: white;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(27, 29, 35);
}
QLineEdit:hover{
	border: 2px solid rgb(64, 71, 88)
}
QLineEdit:focus{
	border: 3px solid rgb(91, 101, 124);
	color: white;
}
QLineEdit:disabled {
	background-color: #808080;
}
/* ToolButton */
QToolButton {
	border: 0px;
	width: 28px;
	height: 28px;
	background-color: transparent;
	color: rgb(255, 255, 255);
	border-radius: 15px;
}
QToolButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QToolButton:pressed {
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}
QPushButton{
	background-color: rgb(27, 29, 35);
}
QPushButton {
	border: 0px solid rgb(52, 59, 72);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	margin-right:10px;
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 5px solid transparent;
}
QPushButton:pressed {
	background-color: rgb(35, 40, 49);
	border: 7px solid transparent;
}
QPushButton:disabled {
	background-color: rgb(102, 109, 122);
	color: rgb(168, 168, 168);
}

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="running_config">
      <attribute name="title">
       <string>Running Configurations</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="environments">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Environments</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="databaseLabel_5">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>File Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDatabasePath_5">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonBrowseDatabase_4">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/icons/folder_icon.png</normaloff>:/icons/icons/folder_icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="databaseLabel_2">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Parent Directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDatabasePath_2">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonBrowseDatabase_3">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/icons/folder_icon.png</normaloff>:/icons/icons/folder_icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Range</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="databaseLabel_4">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDatabasePath_3">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="databaseLabel_3">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>To</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDatabasePath_4">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Binary Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="databaseLabel_7">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDatabasePath_7">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="databaseLabel_6">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>To</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDatabasePath_6">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>211</width>
             <height>41</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Add Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>211</width>
             <height>41</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Add File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="design">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Design</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>658</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>725</width>
            <height>85</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
